### ISSUE TITLE
cut-mix issue2

### ISSUE THREAD

**Issue by DeepHM** [charlesCXK/TorchSemiSeg#60]
Hello.
I'm reading your paper and looking at the code.

First, I know the cutmix method is to mix 2 different images and then input them.

And,
In the cutmix part, you said that different images are called because the sampler is different. (
( # exp.voc/voc8.res50v3%2B.CPS%2BCutMix/dataloader.py  )

but,
( # exp.voc/voc8.res50v3%2B.CPS%2BCutMix/dataloader.py  )

In the code, train_sampler is defined as follows.
-> train_sampler = torch.utils.data.distributed.DistributedSampler(train_dataset)
 the seed of the 'torch.utils.data.distributed.DistributedSampler' is fixed with the default value of 0.
Therefore, two images are created and two images are mixed through different augmentation of the same image.
I think that if you want to synthesize the other two images in the cutmix part, you should call from 'torch.utils.data.distributed.DistributedSampler' to another seed.

Lastly, I want to know if you want to mix two images of the same or two different images in the cut-mix part.

Thank you!

**Comment by charlesCXK** [comment]
Hi, I want to mix two images of **two different images**. I strongly recommend running the code to check whether your guess is correct.

If you define two dataloaders like this, you will get two different loaders.

```python
    sampler_train = torch.utils.data.DistributedSampler(
        dataset_train, num_replicas=num_tasks, rank=sampler_rank, shuffle=True
    )
    sampler_train2 = torch.utils.data.DistributedSampler(
        dataset_train, num_replicas=num_tasks, rank=sampler_rank, shuffle=True
    )

    data_loader_train = torch.utils.data.DataLoader(
        dataset_train, sampler=sampler_train,
        batch_size=args.batch_size,
        num_workers=args.num_workers,
        pin_memory=args.pin_mem,
        drop_last=True,
    )

    data_loader_train2 = torch.utils.data.DataLoader(
        dataset_train, sampler=sampler_train2,
        batch_size=args.batch_size,
        num_workers=args.num_workers,
        pin_memory=args.pin_mem,
        drop_last=True,
    )
```

**Comment by kwakhyunmin** [comment]
Thanks for your kind reply.
